woocommerce/woocommerce
# populate default variation attributes when it is the first variation

### All Submissions:



### Changes proposed in this Pull Request:

When the first variation of a variable product with attributes is created, the attribute meta is not added to the DB. When any subsequent variation is created the attribute meta is added. This PR updates the save to create the attribute meta on the first variation

Closes #22046 .

### How to test the changes in this Pull Request:

1. Create a variable product
2. Add 2+ attributes
3. Add an "Any" variation via Add variation > Go
4. Add a specific variation in the same manner
5. Save, then view product
6. All variation combinations should be selectable.

### Other information:




### Changelog entry

> Fix issue where Any attributes on variable products not always selectable on front end.
